README.md

```diff
@@ -97,7 +97,7 @@ pages might help you get started:
 
 - [Getting started](http://www.freecadweb.org/wiki/Getting_started)
 - [Features list](http://www.freecadweb.org/wiki/Feature_list)
-- [Frequent questions](http://www.freecadweb.org/wiki/FAQ)
+- [Frequent questions](http://www.freecadweb.org/wiki/FAQ/en)
```

Comment:
Thanks for catching this.  
Please change all wiki links to the format below and git force push the commit. 
`https://wiki.freecadweb.org/<wiki page name>`